databases/week2/part3.sql

```diff
@@ -0,0 +1,15 @@
+USE hyf_class2023;
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.id, task.title, user.email FROM task INNER JOIN user ON task.user_id = user.id WHERE user.email LIKE '%@spotify.com';
```

Comment:
Your query is only giving out 1 task where as the user has 4 tasks assigned. My suggestion would be something like this...
```
SELECT user.name,user.email, task.title,task.id
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user ON user.id = user_task.user_id
WHERE user.email LIKE '%@spotify.com';
```

Reply:
same here